docs/development/compare_atomic_files.ipynb

```diff
@@ -6,145 +6,8878 @@
    "source": [
```

Comment:
<p>Color coding for the pct_changes might be useful to spot where the biggest differences are.</p><p> </p><p>Also, in the future it might be nice to have a less detailed overview of the differences in addition to the dataframe above. This summary could for example show for each species how many lines are different and what the maximum differences in the parameters are.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/carsus/pull/315/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>